proof.go

```diff
@@ -144,10 +144,14 @@ func (pln proofLeafNode) Hash() []byte {
 // a path to the least item.
 func (node *Node) PathToLeaf(t *Tree, key []byte) (PathToLeaf, *Node, error) {
 	path := new(PathToLeaf)
-	val, err := node._PathToLeaf(t, key, path)
+	val, err := node._pathToLeaf(t, key, path)
```

Comment:
why underscore in the beginning?

Reply:
Was there before. I think Jae uses them to mark recursive helpers. I can remove it here but there are many other places.